src/librustc_builtin_macros/asm.rs

```diff
@@ -275,6 +275,14 @@ fn err_duplicate_option<'a>(p: &mut Parser<'a>, symbol: Symbol, span: Span) {
         String::new(),
         Applicability::MachineApplicable,
     );
+    if p.look_ahead(0, |t| t == &token::Comma) {
```

Comment:
Why `look_ahead(0...`? This is the same as p.token, I believe.

Reply:
I think I was unsure whether that could cause a panic if the compiler was unexpectedly at the end of the source code. I'm not sure why I thought that though :)